activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -881,7 +881,7 @@ def exec_no_cache(sql, name, binds, async:, allow_retry:, materialize_transactio
 
           type_casted_binds = type_casted_binds(binds)
           log(sql, name, binds, type_casted_binds, async: async) do |notification_payload|
-            with_raw_connection(allow_retry: false, materialize_transactions: materialize_transactions) do |conn|
+            with_raw_connection(allow_retry: allow_retry, materialize_transactions: materialize_transactions) do |conn|
```

Comment:
I suspect passing `allow_retry: false` instead of `allow_retry: allow_retry` in the PG adapter was unintentional, but would like to verify this change _is_ what we want, and we're not opting to explicitly disable retries here.

cc @composerinteralia from https://github.com/rails/rails/pull/49970/commits/aecca3d2d4d0991e974ce9c97d10cb3c87e49ae3

Reply:
🤦🏻 yup, that's definitely what I meant to do in that PR. Thank you!